static/main.js

```diff
@@ -222,20 +229,27 @@ form.addEventListener("submit", (e) => {  // Submit button press event
     stop_button.style.color = "red";
 
     socket.emit("send", dict)
+    init_info_1.innerHTML = '<div style="color:yellow;font-size:50px;font-family: terminal, sans-serif;justify-content: center;align-items: center;display: flex;">Processing started</div>'
 });
 
 
 form.addEventListener("reset", (e) => {  // Stop button press event
     e.preventDefault();
-
-    let stop_button = document.getElementById("stop_button");
-    stop_button.style.color = "black";
-    processing_in_progress = false;
-
-    socket.emit("stop");
-
     let console_element = document.getElementById("init_info");
-    console_element.innerHTML += "Processing stopped\n";
+    if (processing_in_progress) {
+        let init_info_1 = document.getElementById("init_info_1");
+        init_info_1.innerHTML = "";
+        console_element.innerHTML = '<div style="color:red;font-size:50px;">Processing stopped</div>';
+        socket.emit("stop");
+        let stop_button = document.getElementById("stop_button");
+        stop_button.style.color = "black";
+        processing_in_progress = false;
+    } else {
+        let init_info_1 = document.getElementById("init_info_1");
+        init_info_1.innerHTML = "";
+        console_element.innerHTML = '<div style="color:red;font-size:50px;">Error: you haven`t started yet';
```

Comment:
On lines [210](https://github.com/pavtiger/Parse-tables-from-PDF/pull/69/files#diff-acd2228669a20570c1c6bfe003b0b075ded45f1270f2dac8fda906c9583f4531R210) and [202](https://github.com/pavtiger/Parse-tables-from-PDF/pull/69/files#diff-acd2228669a20570c1c6bfe003b0b075ded45f1270f2dac8fda906c9583f4531R202) you changed this message in processing_status box to alert() but here you do the other thing. So to maintain consistency you have to choose one and do all around the code. If you want to do alerts, then do an alert here as well

Reply:
okay, got it.